assets/pages/contributions-landing/newContributionsLanding.jsx

```diff
@@ -87,35 +177,48 @@ const content = (
+                className="form__radio-group__input"
+                type="radio"
+                name="contributionType"
+                value="monthly"
+                checked={contributionType === 'monthly'}
+              />
+              <label htmlFor="contributionType-monthly" className="form__radio-group__label">{selectedCountryGroup.contribution.monthly}</label>
             </li>
             <li className="form__radio-group__item">
-              <input id="contributionType-oneoff" className="form__radio-group__input" type="radio" name="contributionType" value="oneoff" />
-              <label htmlFor="contributionType-oneoff" className="form__radio-group__label">One-off</label>
+              <input
+                id="contributionType-oneoff"
+                className="form__radio-group__input"
+                type="radio"
+                name="contributionType"
+                value="oneoff"
+                checked={contributionType === 'oneoff'}
+              />
+              <label htmlFor="contributionType-oneoff" className="form__radio-group__label">{selectedCountryGroup.contribution.oneoff}</label>
             </li>
           </ul>
         </fieldset>
 
         <fieldset className="form__radio-group form__radio-group--pills form__radio-group--contribution-amount">
           <legend className="form__legend form__legend--radio-group">Amount</legend>
           <ul className="form__radio-group__list">
-            <li className="form__radio-group__item">
-              <input id="contributionAmount-25" className="form__radio-group__input" type="radio" name="contributionAmount" value="25" checked />
-              <label htmlFor="contributionAmount-25" className="form__radio-group__label" aria-label="25 US Dollars">$25</label>
-            </li>
-            <li className="form__radio-group__item">
-              <input id="contributionAmount-50" className="form__radio-group__input" type="radio" name="contributionAmount" value="50" />
-              <label htmlFor="contributionAmount-50" className="form__radio-group__label" aria-label="50 US Dollars">$50</label>
-            </li>
-            <li className="form__radio-group__item">
-              <input id="contributionAmount-100" className="form__radio-group__input" type="radio" name="contributionAmount" value="100" />
-              <label htmlFor="contributionAmount-100" className="form__radio-group__label" aria-label="100 US Dollars">$100</label>
-            </li>
-            <li className="form__radio-group__item">
-              <input id="contributionAmount-250" className="form__radio-group__input" type="radio" name="contributionAmount" value="250" />
-              <label htmlFor="contributionAmount-250" className="form__radio-group__label" aria-label="250 US Dollars">$250</label>
-            </li>
+            {selectedCountryGroup.amounts[contributionType].map((a, i) => (
+              <li className="form__radio-group__item">
```

Comment:
It would be good to name `a` and `i` something more descriptive

Reply:
I think `i` is idiomatic enough but you're right about `a`.